Slider/Slider.js

```diff
@@ -106,6 +106,7 @@ const SliderBase = (props) => {
 	const nativeEventHandlers = useHandlers({
 		onWheel: handle(
 			forProp('disabled', false),
+			forProp('noWheelEvent', false),
```

Comment:
Please check wheel input on a normal slider also.
Since there is no default value for `noWheelEvent`, `forProp` will return `false` in normal sliders. We don't recommend setting a falsy default value, so what about fixing this like below?

```suggestion
			not(forProp('noWheelEvent', true)),
```